mambaSharedFramework/HLSValueTypes.swift

```diff
@@ -130,6 +131,92 @@ public func ==(lhs: HLSEncryptionMethodType, rhs: HLSEncryptionMethodType) -> Bo
     return lhs.type == rhs.type
 }
 
+/// Represents a minimum required HDCP level needed to play content.
+public struct HLSHDCPLevel: Equatable, FailableStringLiteralConvertible {
+    public let type: HDCPLevel
```

Comment:
minor stylistic comment for all these structs here- can we add space between the types and init's? It may help the readability

Reply:
Sure, makes sense, I was really just aiming to keep a similar style to the other enums in this file. I may consider to take the approach of actually using enums for this (like your later comment suggested)... I'm really not sure why this struct approach is taken 🤔

Reply:
Updated here: 34de4a993d2206f793afe18e53db83b07d7f8ca8